### PR TITLE
SAAP-279 Add additional patterns for Java spring in docs

### PR DESCRIPTION
**Pull request by TehreemNisa**
Add additional patterns for Java spring in docs

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-36c27efd`

File: content/sre/logging/logging.md [hunk 1]
```diff
@@ -30,6 +30,17 @@ This will be parsed as follows:
 }
 ```
 
+If the customer wants to parse logs themselves for example application log alerting like shown below
```
Comment (usamaahmadkhan) [inline content/sre/logging/logging.md]: don't use word `customer`.


We need to provide them full steps where to add what so they can configure this properly

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-1566f0c1`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-0b4ac2d1`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-3bef25ad`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-2abfc249`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-eabf8f9b`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-6e3381cd`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-42a407cf`

File: content/sre/logging/logging.md [hunk 2]
```diff
@@ -29,6 +29,46 @@ This will be parsed as follows:
     "correlation": "ef4f3737f2bcf856"
 }
 ```
+)
+
+## Additional config for non JSON log formats
+
+Parsing application logs which are not in `JSON format` can be done using the below scenario.
```
Comment (usamaahmadkhan) [inline content/sre/logging/logging.md]: *can be done as follows:

File: content/sre/logging/logging.md [hunk 3]
```diff
@@ -29,6 +29,46 @@ This will be parsed as follows:
     "correlation": "ef4f3737f2bcf856"
 }
 ```
+)
+
+## Additional config for non JSON log formats
```
Comment (usamaahmadkhan) [inline content/sre/logging/logging.md]: We should make two sections:
 Parse JSON logs
 (above one)
*Parse non JSON logs
 (this one)

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/alerting/log-alerts.md [hunk 4]
```diff
@@ -28,15 +28,7 @@ The configuration to parse/match/send logs can be specified in the [Application
...
-### Additional config for non JSON log formats
-
-Additionally you can parse your application logs if they are not in JSON format by specifying regexes as described below:
-
-| Parameter | Description |
-|:---|:---|
-|.Values.deployment.fluentdConfigAnnotations.regexFirstLine|specify the regex to match the first line of the log|
-|.Values.deployment.fluentdConfigAnnotations.regex|specify the regex to parse the complete log entry|
-|.Values.deployment.fluentdConfigAnnotations.timeFormat|specify the regex to parse time|
+> Note : Additionally you can parse your application logs if they are not in JSON format as specified in this [logging scenario](https://docs.cloud.stakater.com/content/sre/logging/logging.html)
```
Comment (usamaahmadkhan) [inline content/sre/alerting/log-alerts.md]: *If you use a customized format for logs in your application, you can describe your own regex to parse logs. See example in our log parsing section

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-216-d49f3df5`

File: content/sre/alerting/log-alerts.md (hunk 4 above)
Comment (TehreemNisa) [reply content/sre/alerting/log-alerts.md]: resolved

File: content/sre/logging/logging.md (hunk 3 above)
Comment (TehreemNisa) [reply content/sre/logging/logging.md]: updated

File: content/sre/logging/logging.md (hunk 2 above)
Comment (TehreemNisa) [reply content/sre/logging/logging.md]: updated

File: content/sre/logging/logging.md (hunk 1 above)
Comment (TehreemNisa) [reply content/sre/logging/logging.md]: updated

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.155`

**Pull request merged**